### ISSUE TITLE
[Rio de Janeiro] Developer Native na Cyberlabs

### ISSUE THREAD

**Issue by luiguild** [frontendbr/vagas#3122]
Tá afim de trampar num ambiente moderno focado exclusivamente em inovação? Trabalhamos no Rio com tecnologia de ponta para criar aplicações reais de inteligência artificial para navegadores, telefones e clouds próprias com GPU.

Temos uma vaga aberta para Desenvolvedor Native (iOS e Android) Junior ou Pleno que esteja afim de aprender e se juntar ao time que desenvolve o KeyApp. A vaga exige que a pessoa que deseja se candidatar tenha um conhecimento médio/avançado em Kotlin (ou Java), Swift (ou Objective-C), Javascript e Typescript e que tenha tido algum contato com React Native ou NativeScript, que conheça o conceito de componentes e componentização. Da mesma forma, deverá saber os conceitos de responsividade, semântica e estruturação basica de projetos.

Você criará componentes nativos que serão utilizados pelo time de Front-End no nosso aplicativo feito em NativeScript. Um dos componentes que criamos, utiliza a câmera do telefone para fazer reconhecimento facial dos usuários e você trabalhará diretamente com desafios neste estilo.

É importante que a pessoa que deseja se candidatar tenha portfólio aberto e publicado no Github ou equivalente e que tenha tido experiência prévia em outra empresa/startup. Não é necessário estar cursando faculdade na área ou ser formado.

A vaga é para uma pessoa desenvolvedora de software alocada na cidade do Rio de Janeiro, porém, neste momento atual de COVID-19, trabalhos remotos estão sendo considerados.
Gostamos e damos preferência para trocas de experiências no dia a dia, mas temos total flexibilidade para home office e trabalhos a distância.

Somos muito transparentes em tudo que fazemos por aqui e temos o objetivo de manter um ambiente inclusivo e diversificado, desta forma, convidamos candidatas que se identificam como mulheres, pessoas que se identificam como negras, transexuais, homoafetivas e que se enquadram em outras minorias, não nos importamos se você é unicórnio, girafa ou cyborg, literalmente nossas portas estão abertas para todos.

## Local
Temporariamente remoto, Rio de Janeiro - Botafogo | Com horários flexíveis e home office à sua escolha/necessidade

#### Diferenciais
- Os maiores projetos de AI do Brasil
- Ambiente seguro, que preza pela diversidade e inclusão
- Desafios reais de inovação de software
- Ambiente completamente descontraído
- Biblioteca virtual de livros internos da empresa à sua disposição
- Nivelamento de conhecimento periódico dos colaboradores
- Workshops e hackathons internas
- Patrocínio para participação em eventos nacionais

## Requisitos
**Obrigatórios:**
- Alta proficiência em desenvolvimento nativo Android (Java/Kotlin) e/ou iOS (Objective-C/Swift)
- Caso domine apenas uma das plataformas, interesse em aprender a outra
- Escrever código limpo e com boas práticas de engenharia de software
- Capacidade de desenvolver soluções com desempenho otimizado (economia de bateria, processamento e memória)
- Aplicativos publicados na Google Play Store e/ou Apple App Store
- Sede de conhecimento
- Portfólio público
- Git
- Linux

**Desejáveis:**
- Proficiência nas duas plataformas móveis (Android/iOS)
- Conhecimento de Javascript (ES6/Next) e Typescript
- Experiência no desenvolvimento de apps que controlam a câmera
- Experiência no desenvolvimento de bibliotecas e frameworks
- Experiência com as linguagens Kotlin e Swift
- Sólido conhecimento de multithreading
- Github com projetinhos mínimos

**Diferenciais:**
- Ter algum conhecimento sobre Canvas
- Projetos Open Source ❤️ 
- Experiência com Vue.js
- Experiência com NativeScript
- Experiência na utilização de modelos de Deep Learning com Tensorflow Lite ou Pytorch Mobile
- Experiência em C+

## Contratação
O regime de contratação é exclusivamente PJ, temos descanso remunerado, bônus salarial ao final do ano e flexibilidade total de horários.

**Faixa salarial proposta:**
- R$4.000,00 ~ R$10.000,00
(Valor a ser definido em conjunto com a pessoa que desejar se candidatar com base em diversos fatores como comunicação, experiências profissionais anteriores, portifólio e skills diversas)

Possuímos um programa de progressão salarial explícito e transparente onde você terá visibilidade dos próximos passos que você pode dar internamente na empresa.

## Nossa empresa
Se você curte inovação, a Cyberlabs irá lhe proporcionar um ambiente gigante de aprendizado. Hoje somos os maiores patrocinadores de pesquisa e desenvolvimento de AI no Brasil, patrocinando diversas bolsas de mestrado, doutorado e pós-doc, além de termos um centro de treinamento de AI no coração de Botafogo. Já somos 50+ pessoas em constante evolução e crescimento. Temos integrantes no Rio, São Paulo, Goiânia, Espírito Santo e Brasília e projetos grandes de AI no Brasil todo pra você participar.

## Como se candidatar
Se depois de ter lido isso tudo, você continua se interessando no que fazemos por aqui, nós gostaríamos de trabalhar com você! Envie um email para luigui@cyberlabs.ai tendo como assunto: [VAGA] [Rio de Janeiro] Developer Native na Cyberlabs

O conteúdo do e-mail que você nos enviar nos ajudará a te conhecer melhor, então escreva uma breve apresentação sobre você contando o que gosta, o que faz atualmente, algumas curiosidades sobre você e principalmente, não esqueça de colocar os links do seu LinkedIn, GitHub/Gitlab/Bitbucket/Codepen e também como podemos falar com você, seu telefone, Telegram e se possui Whatsapp.

Capriche no texto, um e-mail sem conteúdo ou apenas com o link ou anexo do seu currículo não te ajudará em nada. Escrever sobre você te ajudará a se destacar entre os diversos e-mails que recebemos. Fique tranquilo que responderemos seu e-mail o mais breve possível e te daremos um feedback sobre o que você mandou, seja lá o que for. Sinta-se a vontade para me enviar um e-mail quando quiser ou me chamar no telegram @luiguild.

#### Alocação
- Alocado ou Remoto

#### Regime
- PJ

#### Nível
- Junior
- Pleno

#### Labels
- Remoto
- Alocado
- Junior
- Pleno
- PJ

Estamos aguardando seu contato 😄 ❤️

**Comment by luiguild** [comment]
Up <3

**Comment by felipefialho** [comment]
Esta vaga encontra-se há um bom tempo sem novas interações. Se ainda estiver aberta, faça um comentário, caso contrario, a fecharemos automaticamente em 5 dias.

**Comment by luiguild** [comment]
tá aberta ainda, na real, ninguém nem mandou contato... quem ler e estiver com alguma dúvida, pode entrar em contato comigo \o/
